pkg/morph/client/constructor.go

```diff
@@ -60,6 +63,7 @@ func defaultConfig() *cfg {
 		signer: &transaction.Signer{
 			Scopes: transaction.Global,
 		},
+		switchInterval: defaultSwitchInterval,
```

Comment:
Can we make it `0` (disabled) in `morph/client` and ensure to provide interval in both IR and neofs-node?